nova_plugin/tests/resources/test-networks-relationships-blueprint.yaml

```diff
@@ -0,0 +1,82 @@
+
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - plugin.yaml
```

Comment:
The yaml in the root of the repository, with openstack's types definitions.
I think this works by accident really? This is copied from the other test blueprint, though. We can change both to use some more reliable path?